app/models/furniture.rb

```diff
@@ -83,4 +83,15 @@ def self.from_placement(placement)
   def to_kind_class
     becomes(Furniture.registry.fetch(furniture_kind.to_sym))
   end
+
+  # Adds a writer and a reader for a value backed by `settings`
+  def self.setting(setting_name)
```

Comment:
A V minor nit that I should have brought up yesterday and is more related to dealing with high-allocations in non-Convene Rails applications; but the unboxing of the symbol to a string could happen once at `self.setting` time rather than 2x when calling `setting` time and Nx when calling the setting methods. I'm ashamed that I even think about this now 🤣